plugins/woocommerce-docs/src/Manifest/ManifestProcessor.php

```diff
@@ -20,50 +18,6 @@ public static function process_manifest( $manifest, $logger_action_id ) {
 		self::process_categories( $manifest['categories'], $logger_action_id );
```

Comment:
Haha no, I think I just didn't notice in the clean up. Will fix :D

Reply:
Actually looking back yes this is still intentional. not because of a preparation for the future but because this is a recursive function with optional args that should be controlled internally and not exposed as API to the user of this class.

It's a fairly typical pattern with recursive functions to have them controlled from a top level function call inside the class because you don't want the user of this class to worry about those details.

Reply:
Ah yes, now I see what you mean. That's fair. 

I'm still unsure about calling it `process_manifest` and passing in the whole `$manifest` when we only need and process the categories. 🤔